tests/helper/helper_oc.go

```diff
@@ -593,3 +593,31 @@ func (oc OcRunner) GetAllPVCNames(namespace string) []string {
 func (oc OcRunner) DeletePod(podName string, namespace string) {
 	CmdShouldPass(oc.path, "delete", "pod", "--namespace", namespace, podName)
 }
+
+func (oc OcRunner) GetAllPodsInNs(namespace string) string {
+	cmd := []string{"get", "pods", "-n", namespace}
```

Comment:
This is more like args and not cmd. Since we're passing these as arguments to the oc command a couple of lines later. We're calling it as `ocArgs` at numerous places in the tests.

Reply:
Updated